junit-platform-commons/src/module/org.junit.platform.commons/module-info.java

```diff
@@ -9,7 +9,8 @@
  */
 
 module org.junit.platform.commons {
-	requires java.logging; // TODO Is "requires transitive java.logging" needed here?
```

Comment:
Removed the superseded `TODO` on-the-fly -- the module API of `org.junit.platform.commons` does not use types of `java.logging`.